### ISSUE TITLE
Does nothing when I click download

### ISSUE THREAD

**Issue by ghost** [dekiller82/Spotify-Playlist-Downloader-GUI#6]
The downloader does not do anything when I click download. Nothing shows in the progress bar either. My username has a "-" in it, not sure if that's the problem...

**Comment by ghost** [comment]
It may have something to do with npm because it opens and closes right away every time.

**Comment by ghost** [comment]
1. Check if node is installed propertly.
2. Open the command prompt, get into the program folder and run "npm install". It will install missing node packages.

**Comment by ghost** [comment]
Hi,

I have installed the nodejs properly

I opened cmd, did "cd C:\Program Files\nodejs" to get to the program folder, and then ran npm install

I got this result: https://i.gyazo.com/534378c03ab9fcd2c0b98a367eae01e7.png

Then I ran npm install in C:\Program Files\nodejs\node_modules\npm and I get this: https://i.gyazo.com/dd2d40777bcd577f748b8b537faa0875.png

I ran spotify playlist downloader.exe, and the problem still occurs.

**Comment by ghost** [comment]
You did it in a wrong folder.

When I said "get into the program folder", I mean the Spotify Downloader folder, the directory where you extracted all the files downloaded from Github :+1:

**Comment by ghost** [comment]
Ok. I reinstalled nodejs, redownloaded a fresh copy of the GUI, ran npm install in that folder with all the files, and then ran it. Same problem. This seems to be a problem for others around the web as well.

**Comment by ghost** [comment]
Yes, we all are having the same issue. I hope will be fixed soon.

**Comment by ghost** [comment]
Are a developer of this GUI or just a user?

**Comment by ghost** [comment]
Just an advanced user.

**Comment by FallenArchangel** [comment]
I realize this is pretty old, but is there any chance someone knows how to make this thing log in?  It's trying to log in without doing the capcha.

Also, for some reason nodejs installs to Program Files (x86) even if you use the 64-bit version, which will cause this to crash do to line 291.
